src/metabase/async/streaming_response.clj

```diff
@@ -159,7 +221,13 @@
   ;; async responses only
   compojure.response/Sendable
   (send* [this request respond* _]
-    (respond* (compojure.response/render this request))))
+    (respond* (compojure.response/render this request)))
+
+  ;; TODO -- if we want this to work when running via `lein ring server` we need to add an impl for
```

Comment:
I'm getting `No implementation of method: :write-body-to-stream of protocol: #'ring.core.protocols/StreamableResponseBody found for class: metabase.async.streaming_response.StreamingResponse` when running app with `lein ring server` which is a not good experience :) It I could be nice to implement the protoccol for `metabase.async.streaming-response` namespace.

Reply:
I think everyone on our dev team either lauches with `lein run` (if not making backend changes) or launches the server from the REPL. So use one of those for the time being until I get a chance to fix it. 

`lein ring server` (dev server) is currently not working with the new streaming response stuff because we had to make some tweaks to the Ring server for streaming, and `lein ring server` supplies its own server. So I will have to either write a new impl of `lein ring server` when I get a chance or deprecate it in favor of an auto-reloading version of `lein run`

Note that this is not specific to this PR (it's been the case on `master` for a couple of weeks now, and you're the first one to notice AFAIK)

Reply:
I opened #12181 to remind myself to fix it